tardis/plasma/properties/ion_population.py

```diff
@@ -406,6 +400,12 @@ class IonNumberDensityHeNLTE(ProcessingPlasmaProperty):
     for the electron densities are not similar to within the threshold
     value, a new guess for the value of the electron density is chosen
     and the process is repeated.
+    
+    Attributes
+    ----------
+    ion_number_density : Pandas DataFrame, dtype float
+                         Index atom number, ion number. Columns zones.
+    electron_densities : Numpy Array, dtype float
```

Comment:
this should be `numpy.ndarray` the second line can be comments like dtype

Reply:
Why would numpy.ndarray have the type in a comment, but the pandas.DataFrame/MultIndex etc have it as shown. Wouldn't we want to keep the type descriptor at the top to be consistent?